package.json

```diff
@@ -29,17 +29,16 @@
     "link-win": "node ./scripts/link-bin.js node_modules/amplify-cli-internal/bin/amplify amplify-dev",
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
     "package": "lerna run package",
-    "postpublish:release": "git fetch . release:main && git push origin main",
-    "postpublish:api-plugin-stable": "git push origin api-plugin-stable",
+    "postpublish:release-api-plugin-stable": "./codebuild_specs/scripts/post_publish.sh",
```

Comment:
I've verified that lerna detects these post-publish scripts and runs them after their `publish` equivalents by adding a `postpublish:tag` temporarily. To be extra sure that it doesn't accidentally run `postpublish:release` and do destructive things to `main`, removed that script which is not needed.